test/fixtures/pythagorean-triplet/exemplar/pythagorean-triplet.spec.js

```diff
@@ -11,26 +11,26 @@ describe('Triplet', () => {
     expect(tripletsWithSum(12)).toEqual([[3, 4, 5]]);
   });
 
-  xtest('triplets whose sum is 108', () => {
+  test('triplets whose sum is 108', () => {
```

Comment:
Why was the skip part removed here? Don't we want to check the test runner does the unskipping correctly? Or is this handled differently now?

Reply:
Oh interestingly... This was because I ran the test runner on this file directly 🤣🤣